.github/workflows/require-linked-issue.yaml

```diff
@@ -0,0 +1,19 @@
+name: require linked issue
+
+on:
+  pull_request_target:
+    branches: ["main"]
+    types: [opened, edited, reopened, synchronize]
+
+jobs:
+  check_pull_requests:
+    runs-on: ubuntu-latest
+    name: require linked issue
+    permissions:
+        issues: read
+        pull-requests: read
+    steps:
+      - uses: nearform-actions/github-action-check-linked-issues@v1
```

Comment:
Generally I'm a little suspicious about using (let alone relying on) actions from "some random repo" on the internet, though with only read permissions on what is anyway a public repo, the risk seems pretty small even in the worst case

Reply:
A quick skim through the code makes it look like it's doing exactly what I'd expect, FWIW (though I could easily miss something, and it could easily change in the future)

Reply:
My thoughts also. While their examples required write on PRs (which is fairly benign tbh), I wanted to limit permissions to read only. I did a quick skim of the code but thank for also taking a look.